api/app/controllers/api/v1/users_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
 
       def create
         @user = ::Updaters::User.new(user_params).update(User.new)
-        include_password = user_params[:data][:meta][:created_by_admin] == true
+        include_password = user_params.dig("data", "meta", "created_by_admin") == true
```

Comment:
In the future, you can use the Ruby safe navigation operator instead of `dig`. 

http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/